fix conflict with crhomium patch:
revert 37e4cda28fd3d7a5f4f7ffb1057a3604dba683db

Index: electron/patches/chromium/.patches
--- electron/patches/chromium/.patches.orig
+++ electron/patches/chromium/.patches
@@ -148,5 +148,4 @@ cherry-pick-b922fcb61e3b.patch
 cherry-pick-0d9598145069.patch
 cherry-pick-24329fe5c4d0.patch
 m120-lts_mediasession_use_a_mediasessionimpl_weakptr_in.patch
-x11_fix_crash_when_randr_getmonitors_fails.patch
 feat_enable_passing_exit_code_on_service_process_crash.patch
